tests/behat/fixtures/d11/config/sync/field.field.node.page.field_daterange_date_only.yml

```
@@ -0,0 +1,21 @@
+    - node.type.page
+  module:
+    - datetime_range
+id: node.page.field_daterange_date_only
+field_name: field_daterange_date_only
+entity_type: node
+bundle: page
+label: 'Event period date only'
+description: ''
+required: false
+translatable: false
+default_value: {  }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)